### PR TITLE
taxonomy: Update improvements.txt

### PR DESCRIPTION
**Pull request by ItshMoh**
I have added the Hindi translation of each sentences. These translations are all correct and used in daily life. Kindly merge it. It will be very helpful for hindi speaking people.

### Related issue(s) and discussion

- Fixes #[ISSUE NUMBER]

### CODE REVIEW

**Comment by ItshMoh** [comment]
please merge it. It was my first open source contribution.

File: taxonomies/improvements.txt [hunk 1]
```diff
@@ -2,84 +2,117 @@
 
 en:Nutrition - High sugars value for category
 fr:Nutrition - Haut taux de sucres pour la catégorie
+hi:पोषण      - श्रेणी के लिए अधिक चीनी मात्रा
```
Comment (teolemon) [inline taxonomies/improvements.txt:5]:
```suggestion
hi:पोषण - श्रेणी के लिए अधिक चीनी मात्रा
```

Comment (teolemon) [reply taxonomies/improvements.txt:5]: @ItshMoh can you remove the extra spaces like in my example ?

**Comment by ItshMoh** [comment]
@teolemon i have removed the spaces as you suggested.

File: taxonomies/improvements.txt [hunk 2]
```diff
@@ -2,84 +2,129 @@
...
 fr:Nutrition - Très haut taux d'acides gras saturés pour la catégorie
+hi:पोषण - श्रेणी के लिए बहुत अधिक संतृप्त वसा मात्रा
 
 en:Better Nutri-Score with less sodium
 fr:Meilleur Nutri-Score avec moins de sodium
+hi:कम सोडियम के साथ बेहतर न्यूट्री-स्कोर
+
 description:en:A decrease of sodium by 5% to 10% would result in a better Nutri-Score.
 description:fr:Une réduction du sodium de 5% à 10% entraînerait un meilleur Nutri-Score.
+वर्णन:hi:सोडियम में 5% से 10% की कमी से न्यूट्री-स्कोर बेहतर होगा।
```
Comment (alexgarel) [inline taxonomies/improvements.txt:41]: You have to keep property name in english !

```suggestion
description:hi: सोडियम में 5% से 10% की कमी से न्यूट्री-स्कोर बेहतर होगा।
```

File: taxonomies/improvements.txt [hunk 3]
```diff
@@ -2,84 +2,129 @@
...
 description:fr:Une réduction des sucres de 5% à 10% entraînerait un meilleur Nutri-Score.
+वर्णन:hi:चीनी में 5% से 10% की कमी से न्यूट्री-स्कोर बेहतर होगा।
 
 en:Better Nutri-Score with less saturated fat
 fr:Meilleur Nutri-Score avec moins d'acides gras saturés
+hi:कम संतृप्त वसा के साथ बेहतर न्यूट्री-स्कोर
+
 description:en:A decrease of saturated fat by 5% to 10% would result in a better Nutri-Score.
 description:fr:Une réduction des acides gras saturés de 5% à 10% entraînerait un meilleur Nutri-Score.
+वर्णन:hi:संतृप्त वसा में 5% से 10% की कमी से न्यूट्री-स्कोर बेहतर होगा।
```
Comment (alexgarel) [inline taxonomies/improvements.txt:57]:
```suggestion
description:hi:संतृप्त वसा में 5% से 10% की कमी से न्यूट्री-स्कोर बेहतर होगा।
```

File: taxonomies/improvements.txt [hunk 4]
```diff
@@ -2,84 +2,129 @@
...
 
 en:Better Nutri-Score with more proteins
 fr:Meilleur Nutri-Score avec plus de protéines
+hi:अधिक प्रोटीन के साथ बेहतर न्यूट्री-स्कोर
+
 description:en:An increase of proteins by 5% to 10% would result in a better Nutri-Score.
 description:fr:Une augmentation des protéines de 5% à 10% entraînerait un meilleur Nutri-Score.
+वर्णन:hi:प्रोटीन में 5% से 10% की वृद्धि से न्यूट्री-स्कोर बेहतर होगा।
```
Comment (alexgarel) [inline taxonomies/improvements.txt:65]:
```suggestion
description:hi:प्रोटीन में 5% से 10% की वृद्धि से न्यूट्री-स्कोर बेहतर होगा।
```

File: taxonomies/improvements.txt [hunk 5]
```diff
@@ -2,84 +2,129 @@
...
+
 
 en:Better Nutri-Score with more fiber
 fr:Meilleur Nutri-Score avec plus de fibres
+hi:अधिक फाइबर के साथ बेहतर न्यूट्री-स्कोर
+
 description:en:An increase of fiber by 5% to 10% would result in a better Nutri-Score.
-description:fr:Une augmentation des fibres de 5% à 10% entraînerait un meilleur Nutri-Score.
+description:fr:Une augmentation des fibres de 5% à 10% entraînerait un meilleur Nutri-Score.   
+वर्णन:hi:फाइबर में 5% से 10% की वृद्धि से न्यूट्री-स्कोर बेहतर होगा।
```
Comment (alexgarel) [inline taxonomies/improvements.txt:74]:
```suggestion
description:hi:फाइबर में 5% से 10% की वृद्धि से न्यूट्री-स्कोर बेहतर होगा।
```

File: taxonomies/improvements.txt [hunk 6]
```diff
@@ -2,84 +2,129 @@
...
+
 description:en:An increase of proteins by 5% to 10% would result in a better Nutri-Score.
 description:fr:Une augmentation des protéines de 5% à 10% entraînerait un meilleur Nutri-Score.
+वर्णन:hi:प्रोटीन में 5% से 10% की वृद्धि से न्यूट्री-स्कोर बेहतर होगा।
+
 
 en:Better Nutri-Score with more fiber
 fr:Meilleur Nutri-Score avec plus de fibres
+hi:अधिक फाइबर के साथ बेहतर न्यूट्री-स्कोर
+
```
Comment (alexgarel) [inline taxonomies/improvements.txt:71]: Beware: Don't had new line inside an item definition 

```suggestion
hi:अधिक फाइबर के साथ बेहतर न्यूट्री-स्कोर
```

File: taxonomies/improvements.txt [hunk 7]
```diff
@@ -2,84 +2,129 @@
...
+hi:पोषण - श्रेणी के लिए अधिक संतृप्त वसा मात्रा
 
 en:Nutrition - Very high saturated fat value for category
 fr:Nutrition - Très haut taux d'acides gras saturés pour la catégorie
+hi:पोषण - श्रेणी के लिए बहुत अधिक संतृप्त वसा मात्रा
 
 en:Better Nutri-Score with less sodium
 fr:Meilleur Nutri-Score avec moins de sodium
+hi:कम सोडियम के साथ बेहतर न्यूट्री-स्कोर
+
```
Comment (alexgarel) [inline taxonomies/improvements.txt:38]:
```suggestion
```

File: taxonomies/improvements.txt [hunk 8]
```diff
@@ -2,84 +2,129 @@
...
+hi:कम चीनी के साथ बेहतर न्यूट्री-स्कोर
+
 description:en:A decrease of sugars by 5% to 10% would result in a better Nutri-Score.
 description:fr:Une réduction des sucres de 5% à 10% entraînerait un meilleur Nutri-Score.
+वर्णन:hi:चीनी में 5% से 10% की कमी से न्यूट्री-स्कोर बेहतर होगा।
 
 en:Better Nutri-Score with less saturated fat
 fr:Meilleur Nutri-Score avec moins d'acides gras saturés
+hi:कम संतृप्त वसा के साथ बेहतर न्यूट्री-स्कोर
+
```
Comment (alexgarel) [inline taxonomies/improvements.txt:54]:
```suggestion
```

File: taxonomies/improvements.txt [hunk 9]
```diff
@@ -2,84 +2,129 @@
...
+hi:कम सोडियम के साथ बेहतर न्यूट्री-स्कोर
+
 description:en:A decrease of sodium by 5% to 10% would result in a better Nutri-Score.
 description:fr:Une réduction du sodium de 5% à 10% entraînerait un meilleur Nutri-Score.
+वर्णन:hi:सोडियम में 5% से 10% की कमी से न्यूट्री-स्कोर बेहतर होगा।
 
 en:Better Nutri-Score with less sugars
 fr:Meilleur Nutri-Score avec moins de sucres
+hi:कम चीनी के साथ बेहतर न्यूट्री-स्कोर
+
```
Comment (alexgarel) [inline taxonomies/improvements.txt:46]:
```suggestion
```

File: taxonomies/improvements.txt [hunk 10]
```diff
@@ -2,84 +2,129 @@
...
 description:en:An increase of fiber by 5% to 10% would result in a better Nutri-Score.
-description:fr:Une augmentation des fibres de 5% à 10% entraînerait un meilleur Nutri-Score.
+description:fr:Une augmentation des fibres de 5% à 10% entraînerait un meilleur Nutri-Score.   
+वर्णन:hi:फाइबर में 5% से 10% की वृद्धि से न्यूट्री-स्कोर बेहतर होगा।
 
 en:Better Nutri-Score with more fruits and vegetables
 fr:Meilleur Nutri-Score avec plus de fruits et légumes
+hi:अधिक फलों और सब्जियों के साथ बेहतर न्यूट्री-स्कोर
+
```
Comment (alexgarel) [inline taxonomies/improvements.txt:79]:
```suggestion
```

File: taxonomies/improvements.txt [hunk 11]
```diff
@@ -2,84 +2,129 @@
...
 description:en:An increase of fruits, vegetables, nuts and colza/walnut/olive oils by 5% to 10% would result in a better Nutri-Score.
 description:fr:Une augmentation des fruits, légumes, noix, et huiles de colza/noix/olive de 5% à 10% entraînerait un meilleur Nutri-Score.
+वर्णन:hiफलों और सब्जियों में 5% से 10% की वृद्धि से न्यूट्री-स्कोर बेहतर होगा।
```
Comment (alexgarel) [inline taxonomies/improvements.txt:82]:
```suggestion
description:hiफलों और सब्जियों में 5% से 10% की वृद्धि से न्यूट्री-स्कोर बेहतर होगा।
```

File: taxonomies/improvements.txt [hunk 12]
```diff
@@ -2,84 +2,129 @@
...
 
 en:Better Nutri-Score with slightly less sodium
 fr:Meilleur Nutri-Score avec un peu moins de sodium
+hi:थोड़ा कम सोडियम के साथ बेहतर न्यूट्री-स्कोर
+
```
Comment (alexgarel) [inline taxonomies/improvements.txt:87]:
```suggestion
```

File: taxonomies/improvements.txt [hunk 13]
```diff
@@ -2,84 +2,129 @@
...
 description:en:A decrease of sodium by 5% or less would result in a better Nutri-Score.
 description:fr:Une réduction du sodium de 5% ou moins entraînerait un meilleur Nutri-Score.
+वर्णन:hi:सोडियम में 5% से 10% की कमी से न्यूट्री-स्कोर बेहतर होगा।
 
 en:Better Nutri-Score with slightly less sugars
 fr:Meilleur Nutri-Score avec un peu moins de sucres
+hi:थोड़ा कम चीनी के साथ बेहतर न्यूट्री-स्कोर
+
```
Comment (alexgarel) [inline taxonomies/improvements.txt:95]:
```suggestion
```

File: taxonomies/improvements.txt [hunk 14]
```diff
@@ -2,84 +2,129 @@
...
 description:en:A decrease of sugars by 5% or less would result in a better Nutri-Score.
 description:fr:Une réduction des sucres de 5% ou moins entraînerait un meilleur Nutri-Score.
+वर्णन:hi:चीनी में 5% से 10% की कमी से न्यूट्री-स्कोर बेहतर होगा।
```
Comment (alexgarel) [inline taxonomies/improvements.txt:98]:
```suggestion
description:hi:चीनी में 5% से 10% की कमी से न्यूट्री-स्कोर बेहतर होगा।
```

File: taxonomies/improvements.txt [hunk 15]
```diff
@@ -2,84 +2,129 @@
...
 
 en:Better Nutri-Score with slightly less saturated fat
 fr:Meilleur Nutri-Score avec un peu moins d'acides gras saturés
+hi:थोड़ा कम संतृप्त वसा के साथ बेहतर न्यूट्री-स्कोर
+
```
Comment (alexgarel) [inline taxonomies/improvements.txt:103]:
```suggestion
```

File: taxonomies/improvements.txt [hunk 16]
```diff
@@ -2,84 +2,129 @@
...
 description:en:A decrease of saturated fat by 5% or less would result in a better Nutri-Score.
 description:fr:Une réduction des acides gras saturés de 5% ou moins entraînerait un meilleur Nutri-Score.
+वर्णन:hi:संतृप्त वसा में 5% से 10% की कमी से न्यूट्री-स्कोर बेहतर होगा।
 
 en:Better Nutri-Score with slightly more proteins
 fr:Meilleur Nutri-Score avec un peu plus de protéines
+hi:थोड़ा अधिक प्रोटीन के साथ बेहतर न्यूट्री-स्कोर
+
```
Comment (alexgarel) [inline taxonomies/improvements.txt:111]:
```suggestion
```

File: taxonomies/improvements.txt [hunk 17]
```diff
@@ -2,84 +2,129 @@
...
 description:en:An increase of proteins by 5% or less would result in a better Nutri-Score.
 description:fr:Une augmentation des protéines de 5% ou moins entraînerait un meilleur Nutri-Score.
+वर्णन:hi:प्रोटीन में 5% या उससे कम की वृद्धि से न्यूट्री-स्कोर बेहतर होगा।
```
Comment (alexgarel) [inline taxonomies/improvements.txt:114]:
```suggestion
description:hi:प्रोटीन में 5% या उससे कम की वृद्धि से न्यूट्री-स्कोर बेहतर होगा।
```

File: taxonomies/improvements.txt [hunk 18]
```diff
@@ -2,84 +2,129 @@
...
 
 en:Better Nutri-Score with slightly more fiber
 fr:Meilleur Nutri-Score avec un peu plus de fibres
+hi:थोड़ा अधिक फाइबर के साथ बेहतर न्यूट्री-स्कोर
+
 description:en:An increase of fiber by 5% or less would result in a better Nutri-Score.
 description:fr:Une augmentation des fibres de 5% ou moins entraînerait un meilleur Nutri-Score.
+वर्णन:hi:फाइबर  में 5% या उससे कम की वृद्धि से न्यूट्री-स्कोर बेहतर होगा।
```
Comment (alexgarel) [inline taxonomies/improvements.txt:122]:
```suggestion
description:hi:फाइबर  में 5% या उससे कम की वृद्धि से न्यूट्री-स्कोर बेहतर होगा।
```

File: taxonomies/improvements.txt [hunk 19]
```diff
@@ -2,84 +2,129 @@
...
+hi:थोड़ा अधिक प्रोटीन के साथ बेहतर न्यूट्री-स्कोर
+
 description:en:An increase of proteins by 5% or less would result in a better Nutri-Score.
 description:fr:Une augmentation des protéines de 5% ou moins entraînerait un meilleur Nutri-Score.
+वर्णन:hi:प्रोटीन में 5% या उससे कम की वृद्धि से न्यूट्री-स्कोर बेहतर होगा।
 
 en:Better Nutri-Score with slightly more fiber
 fr:Meilleur Nutri-Score avec un peu plus de fibres
+hi:थोड़ा अधिक फाइबर के साथ बेहतर न्यूट्री-स्कोर
+
```
Comment (alexgarel) [inline taxonomies/improvements.txt:119]:
```suggestion
```

File: taxonomies/improvements.txt [hunk 20]
```diff
@@ -2,84 +2,129 @@
...
 description:en:An increase of fiber by 5% or less would result in a better Nutri-Score.
 description:fr:Une augmentation des fibres de 5% ou moins entraînerait un meilleur Nutri-Score.
+वर्णन:hi:फाइबर  में 5% या उससे कम की वृद्धि से न्यूट्री-स्कोर बेहतर होगा।
 
 en:Better Nutri-Score with slightly more fruits and vegetables
 fr:Meilleur Nutri-Score avec un peu plus de fruits et légumes
+hi:थोड़ा अधिक फलों और सब्जियों के साथ बेहतर न्यूट्री-स्कोर
+
```
Comment (alexgarel) [inline taxonomies/improvements.txt:127]:
```suggestion
```

File: taxonomies/improvements.txt [hunk 21]
```diff
@@ -2,84 +2,129 @@
...
 description:en:An increase of fruits, vegetables, nuts and colza/walnut/olive oils by 5% or less would result in a better Nutri-Score.
 description:fr:Une augmentation des fruits, légumes, noix, et huiles de colza/noix/olive de 5% ou moins entraînerait un meilleur Nutri-Score.
+वर्णन:hi: फलों और सब्जियों  में 5% या उससे कम की वृद्धि से न्यूट्री-स्कोर बेहतर होगा।
```
Comment (alexgarel) [inline taxonomies/improvements.txt:130]:
```suggestion
description:hi: फलों और सब्जियों  में 5% या उससे कम की वृद्धि से न्यूट्री-स्कोर बेहतर होगा।
```

**Review by alexgarel — CHANGES_REQUESTED** [review]
@ItshMoh thank you for your contribution.

But there are some subtleties with taxonomies format:
- you must not had empty line inside an item definition, they are meaningful, a new line starts a new item
- you must not translate properties name

**Comment by alexgarel** [comment]
Please check again your changes after applying my suggestions.

**Comment by alexgarel** [comment]
@ItshMoh can you remove all the empty lines you introduced ? Or do you need more explanations ?

although I now see my comment above has a terrible mistake… it should read:

> you must not add empty line inside an item definition, they are meaningful, a new line starts a new item

So do not separate properties part from the rest by a blank line !

File: taxonomies/improvements.txt [hunk 22]
```diff
@@ -2,84 +2,120 @@
...
+
 description:en:An increase of proteins by 5% to 10% would result in a better Nutri-Score.
 description:fr:Une augmentation des protéines de 5% à 10% entraînerait un meilleur Nutri-Score.
+description:hi:प्रोटीन में 5% से 10% की वृद्धि से न्यूट्री-स्कोर बेहतर होगा।
+
 
 en:Better Nutri-Score with more fiber
 fr:Meilleur Nutri-Score avec plus de fibres
+hi:अधिक फाइबर के साथ बेहतर न्यूट्री-स्कोर
+
```
Comment (benbenben2) [inline taxonomies/improvements.txt:69]:
```suggestion
```

**Comment by benbenben2** [comment]
Hi @alexgarel 
Your suggested changes were applied. Or modified for 1 case.
You can review it, commit suggestion and merge.

**Review by alexgarel — APPROVED** [review]
Thank you so much @ItshMoh !

**Review by alexgarel — CHANGES_REQUESTED** [review]
let me fix it…

**Review by alexgarel — APPROVED** [review]
There were just some more blank line remaining between items and their descriptions. A regexp change fixed that…

**Pull request merged**